apps/openmw/mwgui/tradewindow.hpp

```diff
@@ -65,8 +65,10 @@ namespace MWGui
 
             int mItemToSell;
 
-            int mCurrentBalance;
-            int mCurrentMerchantOffer;
+            bool mIsPlayerSelling;
+            bool mIsMerchantBuying;
```

Comment:
Makes sense. Can you open a new pull request with just these changes? The commit credit should go to you since you originally found the issue. :) Let us know how you want to be listed in AUTHORS.md (nickname, fullname or both)?